idd/Energy+.idd.in

```diff
@@ -84968,6 +84968,12 @@ Coil:UserDefined,
    A2 , \field Overall Model Simulation Program Calling Manager Name
         \type object-list
         \object-list ProgramNames
+        \note This is the name of a program to run that defines the user-defined functionality for this component.
+        \note This can match the name of an EnergyManagementSystem:Program or PythonPlugin:Instance object as defined in the input file.
+        \note For use with the API (Library, Callback) workflow, this field should be the same string that the user
+        \note provides when registering a callback function using the API.  In this workflow, the callback name is not entered
+        \note elsewhere in the IDF, so the object-list will not include this user-specified function identifier.  This will
+        \note cause a warning when editing this field in IDF-Editor.
```

Comment:
So I thought I would need an IDD change, but maybe not.  @JasonGlazer @mjwitte what do you think about a) this text, and b) whether we should adjust any functional changes on this field?

Reply:
Yes to both!  Doing that now!

Reply:
OK, added a bunch of IDD notes to the other user defined types, and explained things in the IO ref.  Here's the current version:

![image](https://github.com/NREL/EnergyPlus/assets/849824/8f2153f4-4696-448a-88b3-cd2228c8b21b)